ipc/cli/src/commands/subnet/create.rs

```diff
@@ -112,11 +118,15 @@ pub struct CreateSubnetArgs {
         long,
         help = "The permission mode for the subnet: collateral, federated and static"
     )]
+    // TODO figure out a way to use a newtype + ValueEnum, or reference PermissionMode::VARIANTS to
```

Comment:
I'm not familiar with `strum`, but there's only a few methods that [ValueEnum](https://docs.rs/clap/4.5.0/clap/trait.ValueEnum.html) needs to implement. Maybe the [derivations](https://github.com/clap-rs/clap/blob/master/clap_derive/src/derives/value_enum.rs) give some clues.